src/gql/graphql-client-api.tsx

```diff
@@ -1451,7 +1451,6 @@ export enum PropertySchemaFieldUi {
   Multiline = 'MULTILINE',
   Range = 'RANGE',
   Selection = 'SELECTION',
-  Slider = 'SLIDER',
```

Comment:
This is a sign your backend isn't up to date, or someone on the backend erased it... I added Slider a week or so ago as a field ui option.
Please doublecheck your backend is on main branch and actually up to date with origin.

Reply:
This might also solve the CI errors